lib/Twig/Environment.php

```diff
@@ -314,8 +325,7 @@ public function getTemplateClass($name, $index = null)
     {
         $key = $this->getLoader()->getCacheKey($name);
         $key .= json_encode(array_keys($this->extensions));
```

Comment:
now that keys are class name, we can. json_encode was used before because no char was forbidden in extension names (we even had a test using a snowman as name at some point to test that), and so we could not implode the array with a separator.